docs/ethereum/rpc-api/methods.md

```diff
@@ -13,6 +13,20 @@ https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_signtransaction
 ## `eth_sendRawTransaction`
 https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_sendrawtransaction
 
+## `eth_subscribe`
+
+Allows for a new subscription to be created. When a subscription is created, `message` events will be fired with new block information.
+
+Unlike the [documentation here](https://ethereum.org/en/developers/tutorials/using-websockets/#eth-subscribe), you can use `eth_subscribe` even with an HTTPS RPC URLs configured in Brave Wallet.
+Only `newHeads` is currently supported.
+
+
+## `eth_unsubscribe`
+
```

Comment:
we should have link like https://docs.infura.io/infura/networks/ethereum/json-rpc-methods/subscription-methods/eth_unsubscribe so users can easily tell what to input as param and the expected output. Same as `eth_subscribe`

Reply:
Added for `eth_unsubscribe` into the commit and force pushed.